core/src/autogluon/core/utils/files.py

```diff
@@ -114,52 +115,6 @@ def check_sha1(filename, sha1_hash):
     return sha1.hexdigest() == sha1_hash
 
 
-def mkdir(path):
-    """Make directory at the specified local path with special error handling.
-    """
-    if len(path) > 0:
-        try:
-            os.makedirs(path)
-        except OSError as exc:  # Python >2.5
-            if exc.errno == errno.EEXIST and os.path.isdir(path):
-                pass
-            else:
-                raise
-
-def raise_num_file(nofile_atleast=4096):
-    try:
-        import resource as res
-    except ImportError: #Windows
-        res = None
-    if res is None:
-        return (None,)*2
-    # what is current ulimit -n setting?
-    soft, ohard = res.getrlimit(res.RLIMIT_NOFILE)
-    hard = ohard
-    # increase limit (soft and even hard) if needed
-    if soft < nofile_atleast:
-        soft = nofile_atleast
-
-        if hard < soft:
-            hard = soft
-
-        #logger.warning('setting soft & hard ulimit -n {} {}'.format(soft,hard))
-        try:
-            res.setrlimit(res.RLIMIT_NOFILE, (soft, hard))
-        except (ValueError, res.error):
-            try:
-               hard = soft
-               logger.warning('trouble with max limit,  retrying with soft, hard {}, {}'.format(soft, hard))
-               res.setrlimit(res.RLIMIT_NOFILE, (soft, hard))
-            except Exception:
-               logger.warning('failed to set ulimit')
-               soft, hard = res.getrlimit(res.RLIMIT_NOFILE)
-
-    return soft, hard
-
-raise_num_file()
```

Comment:
@gradientsky This is an example of this outdated code doing 'weird things' when imported.